Jenkinsfile

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env groovy
+
+pipeline {
+  agent { label 'executor-v2' }
+
+  options {
+    timestamps()
+    buildDiscarder(logRotator(numToKeepStr: '30'))
+  }
+
+  stages {
+    stage('Run tests') {
+      steps {
+        sh './test.sh'
+
+        junit 'features/reports/**/*.xml, spec/reports/*.xml'
+      }
+    }
+
+    // Only publish to RubyGems if the HEAD is
+    // tagged with the same version as in version.rb
+    stage('Publish to RubyGems') {
+      agent { label 'releaser-v2' }
+
+      when {
+        expression { currentBuild.resultIsBetterOrEqualTo('SUCCESS') }
```

Comment:
Can we please add a check on the branch name (should be `master`):
```
...
expression { currentBuild.resultIsBetterOrEqualTo('SUCCESS') }
branch "master"
expression {
  ...
```

Reply:
I must have added this at the same time that you were commenting :)